src/test/javascript/spec/component/iris/settings/iris-common-sub-settings-update.component.spec.ts

```
@@ -139,41 +144,41 @@
-        comp.allIrisModels = mockModels();
+        comp.availableVariants = mockVariants();
         comp.settingsType = IrisSettingsType.EXERCISE;
         fixture.detectChanges();
 
-        comp.inheritAllowedModels = true;
-        comp.onInheritAllowedModelsChange();
-        expect(comp.subSettings!.allowedModels).toBeUndefined();
-        expect(comp.allowedIrisModels).toEqual(comp.getAvailableModels());
+        comp.inheritAllowedVariants = true;
+        comp.onInheritAllowedVariantsChange();
+        expect(comp.subSettings!.allowedVariants).toBeUndefined();
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)